src/routes.ts

```diff
@@ -371,6 +371,20 @@ namespace routes {
 		}
 	});
 
+	export let devGameOverview = new Route<{ gameId: string }>({
+		path: '/games/:gameId/overview',
```

Comment:
How is this route different than `/games/:gameId`? i.e. what's different between `game-overview` and `page-game-dev`.